spdk-sys/build.sh

```diff
@@ -11,12 +11,11 @@ rm libspdk.so
 
 ./configure --enable-debug \
 	--target-arch=nehalem \
-	--disable-tests \
 	--without-isal \
 	--with-iscsi-initiator \
 	--with-crypto \
 	--with-uring \
-	--disable-unit-tests
+	--enable-unit-tests
```

Comment:
just curious why would we want to build the tests when building spdk as dependency of mayastor

Reply:
when building it this way, and if I'm not sure if its a bug in the library I run them sometimes. I can disable them no problem :+1: